src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let resultStr = '';
```

Comment:
1. не сокращай слова
2. не используй слово result
```suggestion
    let joinedString = '';
```

Reply:
будет соблюдено